content/en/security/application_security/enabling/nodejs.md

```diff
@@ -31,18 +31,15 @@ You can monitor application security for Node.js apps running in Docker, Kuberne
 ## Enabling threat detection
 ### Get started
 
-1. **Update your Datadog Node.js library package** to at least version 2.23.0 (for NodeJS 12+) or 3.10.0 (for NodeJS 14+), by running one of these commands:
+1. **Update your Datadog Node.js library package** to at least version 3.10.0 (for NodeJS 14+) or 4.0.0 (for Node 16+) or 5.0.0 (for Node 18+), by running one of these commands:
    ```shell
-   npm install dd-trace@^2.23.0
    npm install dd-trace@^3.10.0
+   npm install dd-trace@^4
+   npm install dd-trace@^5
    ```
```

Comment:
would be great to push to user to use the latest version instead of 3.10.0 for example

Reply:
And reorder also the L34:
```
1. **Update your Datadog Node.js library package** to at least version 5.0.0 (for Node 18+) or 4.0.0 (for Node 16+) or 3.10.0 (for NodeJS 14+), by running one of these commands:
```

Reply:
Perfect!